main.js

```diff
@@ -72,20 +72,37 @@ function displayChoice(choiceType) {
 }
 
 function saveOutfit() {
-  if (currentOutfit.title === undefined) {
-    currentOutfit.setTitle(outfitNameInput.value);
+  debugger;
```

Comment:
You left a debugger in the code,  can you please remove this before the merge

Reply:
I'll go in and fix it!